src/main/java/uk/gov/hmcts/reform/prl/models/caseflags/request/C100FlagDetailRequest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.prl.models.caseflags.request;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.extern.jackson.Jacksonized;
+import uk.gov.hmcts.reform.prl.enums.YesOrNo;
+
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
this 3 annotations we can remove. they will be covered with Data and builder

Reply:
Done